kratos/utilities/constraint_utilities.cpp

```diff
@@ -22,12 +22,50 @@ namespace Kratos
 {
 namespace ConstraintUtilities
 {
+std::size_t NumberOfActiveConstraints(ModelPart& rModelPart)
+{
+    KRATOS_TRY
+
+    std::size_t number_of_active_constraints = 0;
+
+    // The number of constraints
+    const int number_of_constraints = static_cast<int>(rModelPart.MasterSlaveConstraints().size());
+    const auto it_const_begin = rModelPart.MasterSlaveConstraints().begin();
+
+    // Setting to zero the slave dofs
+    #pragma omp parallel
+    {
+        #pragma omp for schedule(guided, 512) reduction(+:number_of_active_constraints)
```

Comment:
```suggestion
        #pragma omp parallel for reduction(+:number_of_active_constraints)
```

also I think you don't need the outer `omp parallel`

Reply:
OK